src/server/config.ts

```diff
@@ -253,11 +260,11 @@ const CLUSTER_TYPES: SupportedType[] = ['druid', 'postgres', 'mysql'];
 
 var settingsLocation: SettingsLocation = null;
 if (serverSettingsFilePath) {
-  settingsLocation = {
-    location: 'local',
-    readOnly: false, // ToDo: this should be true
-    uri: serverSettingsFilePath
-  };
+  if (SERVER_SETTINGS.settingsUri) {
```

Comment:
Just added on to SETTINGS_MANAGER we should talk about read only UI and if we want that